python/cuml/cuml/svm/svc.pyx

```diff
@@ -82,22 +84,22 @@ cdef extern from "cuml/svm/svm_parameter.h" namespace "ML::SVM":
 
 cdef extern from "cuml/svm/svm_model.h" namespace "ML::SVM":
 
-    cdef cppclass SupportStorage[math_t]:
+    cdef cppclass SupportStorage:
         int nnz
-        int* indptr
-        int* indices
-        math_t* data
+        device_buffer* indptr
```

Comment:
Why are these raw pointers? Why doesn't the class own the buffers? 

For that matter, why doesn't the class have a constructor and a destructor (especially if it contains raw pointers!)